test/gibct/containers/Modals.unit.spec.js

```diff
@@ -0,0 +1,17 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import Modals from '../../../src/js/gi/containers/Modals';
+import reducer from '../../../src/js/gi/reducers';
+
+const store = createStore(reducer);
+
+describe('<Modals>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<Modals store={store}/>);
```

Comment:
Oh, didn't notice till now that this is the decorated component. Sure, would like to see it tested as a regular component like the others.